### PR TITLE
tests: use NIT_TESTING_ID in neo4j tests

### PR DESCRIPTION
**Pull request by Morriar**
Signed-off-by: Alexandre Terrasa <alexandre@moz-code.org>

### CODE REVIEW

**Comment by privat** [comment]
This seems promising, there is a lot of conflicts in the other tests, but none in the two modified files.

**Comment by Morriar** [comment]
test this please

File: lib/postgresql/postgres.nit [hunk 1]
```diff
@@ -78,8 +79,22 @@ class Postgres
...
 
+  # Prepares the statement `query` with `stmt_name` to be executed later
+  #
+  # `num_params` specifies the number of parameters expected at the statement
+  # execution.
+  #
+  # See `exec_prepared` for execution.
   fun prepare(stmt_name:String, query:String, num_params: Int):PGResult do return new PGResult(native_connection.prepare(stmt_name, query, num_params))
 
+  # Execute prepared statement named `stmt_name` with `values`
```
Comment (jcbrinfo) [inline lib/postgresql/postgres.nit]: Verb tenses are not consistent across the module.

**Comment by jcbrinfo** [comment]
+1

**Pull request merged**